src/ui/RealmBrowser/AddClassModal/AddClassModal.tsx

```diff
@@ -136,6 +154,29 @@ export const AddClassModal = ({
               </InputGroupAddon>
             </InputGroup>
           </FormGroup>
+          {primaryKey && (
+            <Card>
+              {/* Todo: Wording */}
+              <CardBody>
+                <CardSubtitle>MongoDB Sync:</CardSubtitle>
+                <CardText>
+                  <small>
+                    {
+                      'If this Realm is intended to be synced with MongoDB, ensure to keep the primary key name "_id", with the type "ObjectId".'
+                    }
+                  </small>
+                </CardText>
+                <CardSubtitle>UUID auto generation:</CardSubtitle>
+                <CardText>
+                  <small>
+                    {
+                      'If the primary key is named "uuid" with the type "string", Studio will auto generate a UUID (v4) for new entries.'
+                    }
+                  </small>
+                </CardText>
+              </CardBody>
+            </Card>
+          )}
```

Comment:
I suggest uploading a screenshot of this to the PR description. It makes it easier for others to review the change in context 👍